include/cloudbreak.bash

```diff
@@ -553,7 +552,7 @@ generate-toml-file-for-localdev() {
 
 generate-toml-file-for-localdev-force() {
     declare traefikFile=${1:-traefik.toml}
-    generate-traefik-toml "$CLOUDBREAK_URL" "$PERISCOPE_URL" "$CONSUMPTION_URL" "$DATALAKE_URL" "$ENVIRONMENT_URL" "$REDBEAMS_URL" "$FREEIPA_URL" "http://$THUNDERHEAD_URL" "$CLUSTER_PROXY_URL" "$ENVIRONMENTS2_API_URL" "$DATALAKE_API_URL" "$DISTROX_API_URL" "$JAEGER_HOST" "$CB_LOCAL_DEV_LIST" > "$traefikFile"
+    generate-traefik-toml "$CLOUDBREAK_URL" "$PERISCOPE_URL" "$CONSUMPTION_URL" "$DATALAKE_URL" "$ENVIRONMENT_URL" "$REDBEAMS_URL" "$FREEIPA_URL" "http://$THUNDERHEAD_URL" "$CLUSTER_PROXY_URL" "$ENVIRONMENTS2_API_URL" "$DATALAKE_API_URL" "$DISTROX_API_URL" "" "$CB_LOCAL_DEV_LIST" > "$traefikFile"
```

Comment:
why the "" ? As I see the JAEGER param was removed

Reply:
AuditApiUrl is the 13th argument, I did not find an environment variable already in use for it: 
```
type traefikTomlParams struct {
	CloudbreakURL       string
	PeriscopeURL        string
	ConsumptionURL      string
	DatalakeURL         string
	EnvironmentURL      string
	RedbeamsURL         string
	FreeIpaURL          string
	ThunderheadURL      string
	ClusterProxyURL     string
	Environments2ApiURL string
	DatalakeApiURL      string
	DistroxApiURL       string
	AuditApiURL         string
	LocalDevList        string
}
```
Before my change, the datalake api url was set for it. (Notice the value for the index 10 repeated):
```
params := traefikTomlParams{args[0], args[1], args[2], args[3], args[4], args[5], args[6], args[7], args[8], args[9], args[10], args[10], args[11], args[12], args[13]}
```

Reply:
@doktoric what would be the audit url in this case?